.github/workflows/pypi-publish.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       url: https://pypi.org/p/dm-meltingpot
     permissions:
       id-token: write
-    timeout-minutes: 90
+    timeout-minutes: 10
```

Comment:
revert please (and commit with `git commit --amend --no-edit` to ensure things stay squashed)